src/main/java/blusunrize/immersiveengineering/common/blocks/IEBaseBlock.java

```diff
@@ -174,15 +184,17 @@ public boolean isOpaqueCube(IBlockState state)
 	}*/
 
 	@Override
-	public boolean causesSuffocation(BlockState state)
+	@SuppressWarnings("deprecation")
+	public boolean causesSuffocation(BlockState state, IBlockReader worldIn, BlockPos pos)
 	{
 		return !notNormalBlock;
 	}
 
 	@Override
+	@SuppressWarnings("deprecation")
 	public boolean isNormalCube(BlockState state, IBlockReader world, BlockPos pos)
 	{
-		return notNormalBlock;
+		return notNormalBlock; //TODO review: should this be `!notNormalBlock`?
```

Comment:
You're right, looks like a copy-paste error.

Reply:
Changed in f0088b34.